thirdparty-config/classes/Race-ExRace.conf

```diff
@@ -1,10 +1,35 @@
-#Name must be unique betweem races, classes, and skilltrees
-Name: ExRace
-MenuIcon: "minecraft:gold_ore"
+# More indepth information can be found on the Ore wiki for NT-RPG
+#
+#"Name" must a string and must be unique betweem races, classes, and skilltrees
+"Name": "ExRace"
+#"ClassType" must be a string and must be defined in the settings.conf file
+"ClassType": "Race"
```

Comment:
"ClassType": "Race" should be 
ClassType: Race

Reply:
Fixed